src/app/public-view/components/blog-gallery/components/blog-gallery-article/blog-gallery-article.component.html

```diff
@@ -12,7 +12,7 @@
       <div class="card-body">
         <span class="card-title font-weight-bold">{{blogArticle.title}}</span>
         <p class="card-text">{{blogArticle.description}}</p>
-        <a [href]="'blog/' + blogArticle.id" class="stretched-link"></a>
+        <a (click)="onBlogArticleImageClick()" class="stretched-link"></a>
```

Comment:
kedze sa ta moja cast nemergla, tak este zmen tu prosim ta nazov funkcie na `onBlogArticleClick()` kedze sa to uz vztahuje na cely prispevok a nie na cast s obrazkom